regression-tests/test-results/msvc-2022-c++latest/pure2-concept-definition.cpp.output

```diff
@@ -1 +1,2 @@
 pure2-concept-definition.cpp
+pure2-concept-definition.cpp2(28): error C2607: static assertion failed
```

Comment:
This is the only regression test glitch I noticed ... ?

Reply:
Never noticed, since I don't use MSVC.
Somehow this is failing: `static_assert(!test_nonthrowing_requirements<std::chrono::seconds>);`.